src/components/TodosFilter/TodosFilter.tsx

```diff
@@ -0,0 +1,37 @@
+import React, { useContext } from 'react';
+import classNames from 'classnames';
+
+import { FilterMode } from '../../types/FilterMode';
+import { AppContext } from '../AppContext';
+
+const filterModes = Object.values(FilterMode);
```

Comment:
Consider using 'const' instead of 'let' for filterModes, as its value does not change.